hadoop-ozone/ozone-manager/pom.xml

```diff
@@ -157,6 +157,33 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>jersey-client</artifactId>
     </dependency>
 
+    <!-- Overrider ranger-intg jackson version -->
```

Comment:
Uh oh I noticed the typo in the comment too late. Guess we can fix it next time.